src/content/posts/2025-pyladiescon-cfp-launch.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: "PyLadiesCon 2025: Call for Proposals is Now Open!"
+meta_title: "PyLadiesCon 2025: Call for Proposals is Now Open!"
+description: "PyLadiesCon 2025 is accepting proposals"
+date: 2025-07-11T05:00:00Z
+image: " "
+categories: ["Blog Post",]
+authors: ["Abigail"]
+tags: ["python", "cfp", "pyladies"]
+draft: true
```

Comment:
```suggestion
draft: false
```

It seems like the post doesn't show up on the preview because of the draft is set to true.